scripts/install/functions.sh

```diff
@@ -85,43 +85,44 @@ function create_shell_app {
 
 function add_b2b {
     if [ "${ADD_B2B_LIBS}" = true ] ; then
-        ng add --skip-confirmation @spartacus/organization@${SPARTACUS_VERSION} --interactive false
-        ng add --skip-confirmation @spartacus/checkout@${SPARTACUS_VERSION} --interactive false --features="Checkout-B2B" --features="Checkout-Scheduled-Replenishment"
+        ng add @spartacus/organization@${SPARTACUS_VERSION} --skip-confirmation --no-interactive
+        ng add @spartacus/checkout --skip-confirmation --no-interactive --features="Checkout-B2B" --features="Checkout-Scheduled-Replenishment"
     fi
 }
 
 function add_cdc {
   if [ "$ADD_CDC" = true ] ; then
-        ng add --skip-confirmation @spartacus/cdc@${SPARTACUS_VERSION} --interactive false
+        ng add @spartacus/cdc@${SPARTACUS_VERSION} --skip-confirmation --no-interactive
```

Comment:
Smoke test comment 2:  When running the CSR app directly via `ng serve` it works.  However there are many deprecation warnings like this one:
```
Warning: '/.../apps/csr/node_modules/@spartacus/qualtrics/_index.scss' imports '~bootstrap/scss/_mixins' with a tilde. Usage of '~' in imports is deprecated.
```

Reply:
These warnings are now gone.